tox.ini

```diff
@@ -1,58 +1,79 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
 [tox]
+minversion = 3.18
 envlist =
-    py27,
-    py35,
-    py36,
-    py37,
-    py38,
-    pypy,
-    pypy3,
-    docs,
-    flake8
+    lint
+    py27
+    py35
+    py36
+    py37
+    py38
+    py39
+    py310
+    pypy
+    pypy3
+    docs
+    coverage
 
 [testenv]
+usedevelop = true
 deps =
-    .[test]
-    zope.testrunner
-    coverage
 commands =
-    coverage run -m zope.testrunner --test-path=src {posargs:-vc}
-setenv =
-    COVERAGE_FILE=.coverage.{envname}
+    zope-testrunner --test-path=src {posargs:-vc}
+extras =
+    test
 
-[testenv:coverage-report]
-basepython = python3.6
-deps = coverage
+[testenv:lint]
+basepython = python3
 skip_install = true
+deps =
+    flake8
+    check-manifest
+    check-python-versions >= 0.19.1
+    wheel
 commands =
-    coverage erase
-    coverage combine
-    coverage report -m
-setenv =
-    COVERAGE_FILE=.coverage
-parallel_show_output = true
-depends =
-    py27,
-    py35,
-    py36,
-    py37,
-    py38,
-    pypy,
-    pypy3,
+    flake8 src setup.py
+    check-manifest
+    check-python-versions
 
 [testenv:docs]
-basepython =
-    python3.6
+basepython = python3
+skip_install = false
+extras =
+    docs
+commands_pre =
 commands =
     sphinx-build -b html -d docs/_build/doctrees docs docs/_build/html
-    sphinx-build -b doctest -d docs/_build/doctrees docs docs/_build/doctest
```

Comment:
This line can be kept by setting `with-sphinx-doctests = true` in `.meta.toml`.

Reply:
Done.
I needed to add `zope.app.wsgi` to the `docs` extra to fix those tests.